units/meson-add-wants.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+unitdir="$1"
+target="$2"
+unit="$3"
+
+unitpath="${DESTDIR}${unitdir}/${unit}"
+dir="${DESTDIR}${unitdir}/${target}"
+
+if [[ "$target" == */ ]]; then
```

Comment:
@jpakkane ah, indeed. The .sh scripts were not marked as executable. After doing that, meson executes them directly and the changed shebang is respected.